<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseDialog</class>
 <widget class="QDialog" name="ChooseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkInstall">
       <property name="text">
        <string>+install</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSilent">
       <property name="text">
        <string>+silent</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
